src/party.cpp

```diff
@@ -432,8 +434,9 @@ bool Party::canUseSharedExperience(const Player* player) const
 		}
 	}
 
-	uint32_t minLevel = static_cast<int32_t>(std::ceil((static_cast<float>(highestLevel) * 2) / 3));
+	uint32_t minLevel = static_cast<int32_t>(static_cast<float>(highestLevel * 2 / 3.));
 	if (player->getLevel() < minLevel) {
+		this->hasRequiredLevelToShare = false;
```

Comment:
Okay, I just used it because we would avoid having variables with the same name if it ever happened with this one, which is unlikely.